tests/__init__.py

```diff
@@ -5,3 +5,121 @@
 USERNAME = 'foouser@foo.com'
 AUTH_TOKEN = 'fooauthtoken'
 BEARER_TOKEN = 'bearer token'
+ALERT_WITH_AGENTS={'alert': [ {
```

Comment:
Fix the styling to match something like this
```python
ALERT_WITH_AGENTS = {
    'alert': [
        {
            "active": 0,
            "agents": [
                {
                    "active": 0,

                },
                {
                    "active": 0,
                }
            ],
            "alertId": 111111111,
            "apiLinks": [
                {
                    "rel": "related",
                    "href": "https://api.thousandeyes.com/v6/tests/4443322"
                },
                {
                    "rel": "data",
                    "href": "https://api.thousandeyes.com/v6/web/http-server/4443322"
                }
            ],
            "permalink": "https://app.thousandeyes.com/alerts/list/?__a=333666&alertId=111111111",
            "severity": "INFO"
        }
    ]
}
```

Reply:
fixed b8a3076